packages/angular/src/schematics/init/schema.json

```diff
@@ -1,6 +1,8 @@
 {
   "$schema": "http://json-schema.org/schema",
   "$id": "SchematicsAngularModuleInit",
+  "id": "SchematicsAngularModuleInit",
```

Comment:
Why is the `id` added?

Reply:
Other Nx Devkit schema's had `id` rather than `$id`.

I wasn't entirely sure if there was a difference.

Reply:
Really? They shouldn't, `$id` is what's supported by recent versions of the JSON Schema specs. During the Angular 12 migration, I'm pretty sure I changed all of them to use `$id` 🤔

Reply:
I can make the change :)